diesel_derives/src/as_expression.rs

```diff
@@ -6,7 +6,7 @@ use util::*;
 pub fn derive(item: syn::DeriveInput) -> Tokens {
     let item_name = item.ident.as_ref().to_uppercase();
     let is_sized = !flag_present(&item.attrs, "not_sized");
-    let sql_types = item.attrs
```

Comment:
Why rename this variable? The old name was better :wink:

Reply:
`sed -i s/types/sql_types/` probably :D

Reply:
Lol whoops